apps/src/music/redux/musicRedux.ts

```diff
@@ -28,6 +29,8 @@ interface MusicState {
   showInstructions: boolean;
   instructionsPosition: InstructionsPosition;
   isBeatPadShowing: boolean;
+  playbackEvents: PlaybackEvent[];
+  lastMeasure: number;
```

Comment:
Should we start adding comments to all of these members?

Reply:
Yeah good call, will add!